ocsf-import/package.yaml

```diff
@@ -0,0 +1,591 @@
+id: ocsf
+name: OCSF
+author: Tenzir
+author_icon: https://github.com/tenzir.png
+description: |
+  A package that handles onboarding of OCSF mapping pipeline topics.
+  Based on OCSF version 1.3.0.
+  See also: https://schema.ocsf.io/1.3.0/
+
+pipelines:
+  # System Activity (1)
+  import-oscf-filesystem-activity:
+    name: "Import OCSF File System Activity"
```

Comment:
None of these pipelines have a description. There isn't much to describe here anyways, so how do we want to deal with that? Just leaving it out?

cc @mavam

Reply:
All pipelines should have a description so that our users get a human-readable version of the TQL.

Reply:
I added descriptions in https://github.com/tenzir/library/pull/10/commits/17ce6ad438e31a5dfd082898001fa3369f2b21a9